src/lib.rs

```diff
@@ -178,11 +180,18 @@ impl Builder {
         BlockingClient::from_builder(self)
     }
 
-    // Build an asynchronous client from builder
-    #[cfg(feature = "async")]
+    /// Build an asynchronous client from builder
+    #[cfg(feature = "tokio")]
     pub fn build_async(self) -> Result<AsyncClient, Error> {
         AsyncClient::from_builder(self)
     }
```

Comment:
IMHO it should be `cfg(all(feature = "async", feature = "tokio"))]` instead, and the tokio feature should not enable the `async` one in `Cargo.toml`.